go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/pelletier/go-toml/v2
 
 go 1.16
 
-require github.com/stretchr/testify v1.8.4
+require (
+	github.com/stretchr/testify v1.8.4
+	github.com/toml-lang/toml-test v1.4.1-0.20231018175634-23c30e9eb082
```

Comment:
I would really like to avoid adding dependencies to go-toml. That's the reason why zip files were used in the first place. I think two options are either still download zips and filter out tests for TOML > 1.0, or create a submodule for the test generator with its own `go.mod`, to avoid imposing those dependencies on users of go-toml.

Reply:
In that case, I would probably suggest doing the latter. I'll update the PR.

Reply:
Unless you're fine with an untagged version. In that case it might be even simpler.